File: i2f-jdk/i2f-swl/web/i2f-swl/src/i2f-turbo-web/i2f-swl/extension/impl/sm/antherd/supplier/SwlAntherdSm4SymmetricEncryptorSupplier.js
import SwlAntherdSm4SymmetricEncryptor from "../SwlAntherdSm4SymmetricEncryptor";

/**
 * @return {SwlAntherdSm4SymmetricEncryptorSupplier}
 * @implements ISwlSymmetricEncryptorSupplier
 */
function SwlAntherdSm4SymmetricEncryptorSupplier() {

}

/**
 * @return {ISwlSymmetricEncryptor}
 */
SwlAntherdSm4SymmetricEncryptorSupplier.prototype.get = function () {
    return new SwlAntherdSm4SymmetricEncryptor()
}

export default SwlAntherdSm4SymmetricEncryptorSupplier

File: i2f-extension/i2f-extension-swl/src/main/java/i2f/extension/swl/impl/sm/antherd/supplier/SwlAntherdSm4SymmetricEncryptorSupplier.java
package i2f.extension.swl.impl.sm.antherd.supplier;

import i2f.extension.swl.impl.sm.antherd.SwlAntherdSm4SymmetricEncryptor;
import i2f.swl.std.ISwlSymmetricEncryptor;
import i2f.swl.std.supplier.ISwlSymmetricEncryptorSupplier;

/**
 * @author Ice2Faith
 * @date 2024/7/10 9:58
 * @desc
 */
public class SwlAntherdSm4SymmetricEncryptorSupplier implements ISwlSymmetricEncryptorSupplier {

    @Override
    public ISwlSymmetricEncryptor get() {
        return new SwlAntherdSm4SymmetricEncryptor();
    }
}
